server/lib/queries.js

```diff
@@ -114,13 +104,13 @@ const getTotalAnnualBudgetForHost = HostCollectiveId => {
     .then(res => Math.round(parseInt(res[0].yearlyIncome, 10)));
 };
 
-const getTotalAnnualBudget = () => {
+const getTotalAnnualBudget = async () => {
   return sequelize
     .query(
       `
   SELECT
     (SELECT
-      COALESCE(SUM(${generateFXConversionSQL()} * 12),0)
+      COALESCE(SUM(${await generateFXConversionSQL()} * 12),0)
```

Comment:
Can you move the await on top, put the value in a variable and use the variable here instead?

Reply:
Understood.